lib/filterx/filterx-error.c

```diff
@@ -101,7 +103,8 @@ filterx_falsy_error_set_values(FilterXError *error, const gchar *message, Filter
 void
 filterx_error_set_info(FilterXError *error, const gchar *info)
 {
-  g_strlcpy(error->info, info, FILTERX_ERROR_INFO_LEN);
+  error->info = (gchar *) info;
+  error->free_info = FALSE;
```

Comment:
Originally `filterx_eval_push_error_info()` and `filterx_error_set_info()` has a bool parameter free_info.

Now that we don't have those, can we rename functions that operate on `free_info = FALSE` objects to make it clear that the info is not owned and has to live long enough? (probably static strings)

Reply:
set_static_info?
filterx_eval_push_error_static_info?

(or anything similar)